src/Modules/Admin/Commands.Admin/GetPowerBIActivityEvents.cs

```diff
@@ -0,0 +1,124 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Net;
+using Microsoft.PowerBI.Common.Api.ActivityEvent;
+using Microsoft.PowerBI.Common.Client;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.PowerBI.Commands.Admin
+{
+    [Cmdlet(CmdletVerb, CmdletName)]
+    [OutputType(typeof(List<object>))]
+    public class GetPowerBIActivityEvents : PowerBIClientCmdlet
+    {
+        public const string CmdletName = "PowerBIActivityEvents";
+        public const string CmdletVerb = VerbsCommon.Get;
+        private bool validationError = false;
+
+        public GetPowerBIActivityEvents() : base() { }
+
+        public GetPowerBIActivityEvents(IPowerBIClientCmdletInitFactory init) : base(init) { }
+
+        [Parameter(Mandatory = true)]
+        public string StartDateTime { get; set; }
+
+        [Parameter(Mandatory = true)]
+        public string EndDateTime { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public string ActivityType { get; set; }
```

Comment:
Will ActivityTypes grow over time or is it there a well defined set?
Just wondering if we should create an enum for this but if the list is ever-growing then I wouldn't

Reply:
It could be ever-growing. I just added one for the auditing for this new API :)